string                      id
string                      agent_name
mac_ros_bridge/Position     pos
string                      tasks
bool                        active